src/content/1.4/kleisli-categories.tex

```diff
@@ -3,8 +3,8 @@
 \lettrine[lhang=0.17]{Y}{ou've seen how to model} types and pure functions as a category. I also
 mentioned that there is a way to model side effects, or non-pure
 functions, in category theory. Let's have a look at one such example:
-functions that log or trace their execution. Something that, in an
-imperative language, would likely be implemented by mutating some global
+functions that log or trace their execution. In an imperative language,
+such functions would likely be implemented by mutating some global
```

Comment:
Is this change intentional? We agree at some point to keep original wording of the book.

Reply:
Yes, I thought the original wording needed improvement... but if you think we shouldn't mess with the original text, I can revert.  Lmkwyt.  Thanks.

Reply:
Let's keep the changes. Ultimately it's up to reviewers of the entire translation. As we want to make a huuuge PR at the end :)